### PR TITLE
terra-brand-footer - added deprecation badge and end of support notice to BrandFooter.1.doc.mdx

### PR DESCRIPTION
**Pull request by mjpalazzo**
Added deprecation badge and end of support notice.

### Summary

**What was changed:**
added deprecation badge and end of support notice to BrandFooter.1.doc.mdx

**Why it was changed:**
communicate to consumers that terra-brand-footer is no longer supported.

### Testing

View About page and confirm that badge and notice display as required

This change was tested using:

- [ ] WDIO
- [ ] Jest
- [ x] Visual testing (please attach a screenshot or recording)
- [ ] Other (please describe below)
- [ ] No tests are needed

### Reviews

In addition to engineering reviews, this PR needs:

- [ ] UX review
- [ ] Accessibility review
- [ ] Functional review
- [x] tech writer review

### Additional Details

**This PR resolves:**

UXPLATFORM-XXXX 

---

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-framework-docs/src/terra-dev-site/doc/brand-footer/BrandFooter.1.doc.mdx [hunk 1]
```diff
@@ -7,8 +8,16 @@ import VerticalBrandFooter from './example/VerticalBrandFooter?dev-site-example'
 
 <Badge />
 
+[![Deprecated](https://badgen.net/badge/status/Deprecated/grey)](https://github.com/cerner/terra-framework/tree/main/packages/(https://github.com/cerner/terra-framework/tree/main/packages/terra-brand-footer#readme)
```
[the remark this answers is not in the document]
Comment (mjpalazzo) [reply packages/terra-framework-docs/src/terra-dev-site/doc/brand-footer/BrandFooter.1.doc.mdx]: Thanks for fixing that. I was also looking into how to fix it. Let me know if I did something to muck it up.

**Review by sycombs — APPROVED** [review]

**Review by adoroshk — APPROVED** [review]
1. Banner looks as expected.
2. Changelog file for `terra-framework-docs` updated as expected.

Screenshot 2024-01-11 at 4 53 22 PM

**Review by RachaelMarso — APPROVED** [review]
Reviewed and approved for OHAIUAD-206120

**Comment by adoroshk** [comment]
I resolved the CHANGELOG conflict created by January 12 release: https://github.com/cerner/terra-framework/pull/1971/commits/a5882f409f12a22f9342a8c986fd5edaea7386de

**Comment by mjpalazzo** [comment]
> Reviewed and approved for OHAIUAD-206120

Thanks, Rachel!

**Pull request merged**